graphs/fleurys_algorithm.py

```diff
@@ -0,0 +1,110 @@
+from collections import defaultdict
+
+
+class Graph:
+
```

Comment:
```suggestion
```

Reply:
Let's get the tests to be green.

Reply:
@cclauss thanks mate.One empty line make the difference.So how is this pull request is labeled as hacktoberfest-accepted